.github/workflows/ci-kafka.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       run: bash scripts/kafka-integration-test.sh -k
 
     - name: Output Kafka logs
-      run: docker logs "${{ job.services.kafka.id }}"
+      run: docker logs kafka
```

Comment:
not related to this PR, but CI was failing repeatedly and no logs were printed. Using `job.services.kafka.id` is a bug, leftover from when kafka was started as part of the workflow, instead of manually.